src/internal_events/aws_kinesis_firehose.rs

```diff
@@ -4,6 +4,36 @@ use vector_core::internal_event::InternalEvent;
 
 use crate::sources::aws_kinesis_firehose::Compression;
 
+#[derive(Debug)]
+pub struct AwsKinesisFirehoseEventsReceived<'a> {
```

Comment:
I'm also not seeing a reference to this event.

Reply:
Weird, not sure how Clippy missed this both locally and in CI... but you're right, it's not used anymore. I'll remove it.